src/app/app-routing.module.ts

```diff
@@ -51,6 +52,14 @@ export const routes: Routes = [
   },
   // Profile
   {
+    path: '_profile',
+    resolve: {
+      profileNavigation: ProfileResolver
+    },
+    loadChildren: './home/home.module#HomeModule'
```

Comment:
According to https://github.com/openshiftio/openshift.io/issues/662 shouldn't this point to the profile update page?

Reply:
No - the resolver redirects, the loadChildren is just a required value by the router.

Reply:
As i read more, I thought that might be the case